tests/Unit/CommercialsPageTest.php

```
@@ -0,0 +1,218 @@
+            }
+        );
+        Functions\when('add_settings_error')->justReturn(null);
+
+        try {
+            self::callPrivate(CommercialsPage::class, 'handle_save');
+        } finally {
+            $_POST = $previous_post;
+        }
+
+        $this->assertSame('', $updates['teksttv_campaigns'][0]['commercial_block_id']);
+    }
```

PHPMD (2.15.0)

[error] test_save_clears_campaign_reference_to_deleted_commercial_block accesses the super-global variable $_POST. (undefined)

(Superglobals)